app/javascript/dashboard/store/modules/reports.js

```diff
@@ -9,6 +10,17 @@ import liveReports from '../../api/liveReports';
 
 const state = {
   fetchingStatus: false,
+  summaryFetchingStatus: {
+    conversations_count: STATUS.FINISHED,
+    incoming_messages_count: STATUS.FINISHED,
+    outgoing_messages_count: STATUS.FINISHED,
+    avg_first_response_time: STATUS.FINISHED,
+    avg_resolution_time: STATUS.FINISHED,
+    resolutions_count: STATUS.FINISHED,
+    bot_resolutions_count: STATUS.FINISHED,
+    bot_handoffs_count: STATUS.FINISHED,
+    reply_time: STATUS.FINISHED,
+  },
```

Comment:
Isn't this a bit overkill? We just need one status for the API call, right?

Reply:
Yeah, makes sense, I can change it.